source/administration/config-settings.rst

```diff
@@ -2995,6 +2997,30 @@ This setting determines whether team leave/join system messages are posted in th
 | This feature's ``config.json`` setting is ``"EnableDefaultChannelLeaveJoinMessages": true`` with options ``true`` and ``false`` for above settings respectively.     |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Allow Cookies for Subdomains
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This setting set's the Domain parameter on our cookies, which allows the browser to send the cookies to subdomains as well.
+
+**True** True when cookies are allowed.
+
+**False** False when cookies are not allowed.
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"AllowCookiesForSubdomains": true`` with options ``true`` and ``false`` for above settings respectively.                 |
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Websocket URL
```

Comment:
This should be written similar to the Site URL field (it's actually a string input, not a true/false field). https://docs.mattermost.com/administration/config-settings.html#site-url

Reply:
Should "Subdomains" be hyphenated (i.e. "Sub-domains")?

Reply:
@lindy65 In this case no, the config setting is written without the hyphen (based on [default.json file](https://github.com/mattermost/mattermost-server/blob/master/config/default.json)).